### PR TITLE
fix(cache): Remove memcache service and references to it

### PR DESCRIPTION
**Pull request by vbudhram**
## Because

- We are not longer using Memcache

## This pull request

- Removes all references to Memcache services

## Issue that this pull request solves

Closes: https://mozilla-hub.atlassian.net/browse/FXA-9356

## Checklist

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: backstage/redis.resources.yaml [hunk 1]
```diff
@@ -22,3 +22,15 @@ spec:
   type: cache
   owner: fxa-devs
   system: mozilla-accounts
+---
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: fxa-customs-cache
+  description: Caches rate limits and IP reuptation scores.
```
Comment (pdehaan) [inline backstage/redis.resources.yaml:30]:
```suggestion
  description: Caches rate limits and IP reputation scores.
```

**Comment by pdehaan** [comment]
Looks like a few instances (7?) within the libs/shared/contentful/src/** dir (which might be valid for unrelated SubPlat Things(tm) since they also mention GraphQL), and the rest of the instances of memcache are in the fxa-customs-server's /scripts/** directory, but all are commented out.

<details>
<summary><kbd>git grep -in "MeMcAcHe" -- ":(exclude)yarn.lock" | cat</kbd></summary>

```js
libs/shared/contentful/src/lib/contentful.client.config.ts:27:  public readonly memCacheTTL?: number;
libs/shared/contentful/src/lib/contentful.client.ts:51:  private graphqlMemCache: Record<string, unknown> = {};
libs/shared/contentful/src/lib/contentful.client.ts:101:    if (this.graphqlMemCache[cacheKey]) {
libs/shared/contentful/src/lib/contentful.client.ts:108:      return this.graphqlMemCache[cacheKey] as Result;
libs/shared/contentful/src/lib/contentful.client.ts:124:      this.graphqlMemCache[cacheKey] = response;
libs/shared/contentful/src/lib/contentful.client.ts:201:      this.contentfulClientConfig.memCacheTTL || DEFAULT_MEM_CACHE_TTL * 1000;
libs/shared/contentful/src/lib/contentful.client.ts:205:      this.graphqlMemCache = {};
packages/fxa-customs-server/scripts/block-ip.js:18:// var Memcached = require('memcached');
packages/fxa-customs-server/scripts/block-ip.js:20:// P.promisifyAll(Memcached.prototype);
packages/fxa-customs-server/scripts/block-ip.js:23://   var usage = 'Usage: block-ip.js <memcacheHost:port> <ip> <seconds>';
packages/fxa-customs-server/scripts/block-ip.js:45:// var mc = new Memcached(process.argv[2], { namespace: 'fxa~' });
packages/fxa-customs-server/scripts/config-list.js:9: in memcache. config-list.js reads the settings from the memcache server
packages/fxa-customs-server/scripts/config-list.js:21:// var Memcached = require('memcached');
packages/fxa-customs-server/scripts/config-list.js:24:// P.promisifyAll(Memcached.prototype);
packages/fxa-customs-server/scripts/config-list.js:27://   var usage = 'Usage: config-list.js <memcacheHost:port>';
packages/fxa-customs-server/scripts/config-list.js:38:// var mc = new Memcached(process.argv[2], { namespace: 'fxa~' });
packages/fxa-customs-server/scripts/config-set.js:9: in memcache. config-set.js reads json from stdin and writes it to the
packages/fxa-customs-server/scripts/config-set.js:10: memcache server specified by the first argument.
packages/fxa-customs-server/scripts/config-set.js:21:// var Memcached = require('memcached');
packages/fxa-customs-server/scripts/config-set.js:27:// P.promisifyAll(Memcached.prototype);
packages/fxa-customs-server/scripts/config-set.js:30://   var usage = 'Usage: cat settings.json | config-set.js <memcacheHost:port>';
packages/fxa-customs-server/scripts/config-set.js:75://   var mc = new Memcached(process.argv[2], { namespace: 'fxa~' });
packages/fxa-customs-server/scripts/customs-info.js:19:// var Memcached = require('memcached');
packages/fxa-customs-server/scripts/customs-info.js:22://   var usage = 'Usage: customs-info.js <memcacheHost:port> <key>';
```
</details>

File: packages/fxa-customs-server/scripts/block-ip.js
Comment (clouserw) [inline packages/fxa-customs-server/scripts/block-ip.js:1]: Why are we commenting this out instead of deleting it?  (same question for the other scripts)

Comment (vbudhram) [reply packages/fxa-customs-server/scripts/block-ip.js:1]: I was thinking the scripts would be updated here, but from triage we will add functionality to admin panel, filed https://mozilla-hub.atlassian.net/browse/FXA-9459. I'll delete these files.

File: packages/fxa-auth-server/README.md [hunk 2]
```diff
@@ -80,8 +80,7 @@ Executing tests using remote databases (MySQL, Redis, Memcached) is possible by
   - REDIS_HOST
   - ACCESS_TOKEN_REDIS_HOST
   - REFRESH_TOKEN_REDIS_HOST
-- Memcached:
-  - MEMCACHE_METRICS_CONTEXT_ADDRESS
+  - METRICS_CONTEXT_ADDRESS
```
Comment (chenba) [inline packages/fxa-auth-server/README.md:83]: I'm not sure what this is.  Was it from a search and delete the memcache prefix?

**Review by chenba — APPROVED** [review]

File: packages/fxa-auth-server/README.md (hunk 2 above)
Comment (vbudhram) [reply packages/fxa-auth-server/README.md:83]: Oops yea it was, should be `METRICS_REDIS_HOST`. I'll update and then merge.

**Pull request merged**